src/cli/dos_cmds.c

```diff
@@ -142,6 +143,88 @@ short cmd_del(short screen, int argc, char * argv[]) {
     }
 }
 
+short cmd_copy(short screen, int argc, char * argv[]) {
+    FRESULT find_result;
+    FRESULT result;
+    DIR dir;         /* Directory object */
+    FILINFO src_info;    /* File information */
+    FILINFO dst_info;
+    FIL src_file;
+    FIL dst_file;
+
+    BYTE buffer[4096];   /* File copy buffer */
+    UINT br, bw;         /* File read/write count */
+
+    char path[MAX_PATH_LEN];
+
+    bool is_directory = false;
+    bool is_append_file = false; 
+
+    TRACE("cmd_copy");
+
+    if (argc > 2) {
+        strcpy(path, argv[2]);
+
+        result = f_stat(argv[2], &dst_info);
+        if (result == FR_OK) {
+            is_directory = dst_info.fattrib & AM_DIR;
+        } else if (result == FR_NO_FILE) {
+            is_directory = false;
+        } else {            
+            goto error;
+        }
+
+        find_result = f_findfirst(&dir, &src_info, "", argv[1]);
```

Comment:
There is a known issue if the source file has a directory prefix. I think I may need to spit the filename on the last Slash and pass in the prefix instead of blank string. Will experiment in future PR.